src/system/Form/RadioBoxGroup.js

```diff
@@ -202,6 +256,7 @@ RadioBoxGroup.propTypes = {
 	errorMessage: PropTypes.string,
 	hasError: PropTypes.bool,
 	required: PropTypes.bool,
+	variant: PropTypes.oneOf( [ 'default', 'chip' ] ),
```

Comment:
@djalmaaraujo 
This prop is not surfacing in the documentation. Any idea why?

Reply:
@aswasif007 If I understood your question.. Maybe because RadBoxGroup is a js file?

Reply:
Then how do the existing props surface 🤔